Changes presentator

LukeDDC opened the pull request:
## Goals
- Creation of an auxiliar class to format the changes generated by codelog..
## Changes
- Creation of VersionDatePresenter class and its unit tests. 
- Method #changes in changelog.rb will use VersionDatePresenter.
- Tests  for the class were updated.

Commit:
modified:   lib/codelog.rb
	modified:   lib/codelog/config.rb
	new file:   lib/codelog/presenters/version_data_presentator.rb
Added VersionDataPresentator and applied it to lib/codelog.rb.
Changed the data_output_format to public in config.rb.

Commit:
modified:   changelogs/changelog_template.md.erb
	modified:   lib/codelog/command/step/changelog.rb
Changed the changelog.rb to use the DateVersionPresenter class.
Changed the changelog_template to use the methods avaliable by the presenter.

Commit:
modified:   lib/fixtures/changelog_template.md.erb
	modified:   spec/codelog/command/step/changelog_spec.rb
Changed the fixture to introduce VersionDataPresenter methods.

Commit:
modified:   lib/codelog.rb
	renamed:    lib/codelog/presenters/version_data_presentator.rb -> lib/codelog/presenters/version_data_presenter.rb
Changed the name of the file to match the name of the class.

Commit:
new file:   spec/codelog/presenters/version_data_presenter.rb
Added unit test for VersionDataPresenter class.

Commit:
Merge branch 'new_erb_template_file' of github.com:felipe-michelon/codelog into changes-presentator

Commit:
renamed:    spec/codelog/presenters/version_data_presenter.rb -> spec/codelog/presenters/version_data_presenter_spec.rb
Renamed the test.

Commit:
Merge branch 'new_erb_template_file' of github.com:felipe-michelon/codelog into changes-presentator

Commit:
new file:   changelogs/unreleased/20180206141658318_change.yml

michelonfelipe commented on lib/codelog/presenters/version_data_presenter.rb:
```diff
@@ -0,0 +1,21 @@
+module Codelog
+  module Presenters
```
Module name should be singular

michelonfelipe commented on lib/codelog/presenters/version_data_presenter.rb:
```diff
@@ -0,0 +1,21 @@
+      end
+
+      def modifications
+        @changes.dup.delete_if { |key| key == 'Version' || key == 'Date' }
```
It should be better if you add the keys that need to be excluded in a array, and use 
`delete_if { |key| exclusion_array.include?(key) }`

Commit:
modified:   lib/codelog/command/step/changelog.rb
	modified:   lib/codelog/presenters/version_data_presenter.rb
	modified:   spec/codelog/command/step/changelog_spec.rb
	modified:   spec/codelog/presenters/version_data_presenter_spec.rb
Changed the name of the Presenters modulo to singular.
Changed the modifications method in VersionDataPresenter.
